modelbaker/ilitoppingmaker/ili2dbsettings.py

```diff
@@ -0,0 +1,100 @@
+# -*- coding: utf-8 -*-
+"""
+/***************************************************************************
+                              -------------------
+        begin                : 2022-07-17
+        git sha              : :%H$
+        copyright            : (C) 2022 by Dave Signer
+        email                : david at opengis ch
+ ***************************************************************************/
+
+/***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+"""
+
+
+class Ili2dbSettings(dict):
+    def __init__(self):
+        self.parameters = {}
+        self.metaattr_path = None
+        self.postscript_path = None
+        self.prescript_path = None
+        self.models = []
+
+    def parse_parameters_from_db(self, db_connector):
+        """
+        We do only care about the settings "known" by model baker (set by user in GUI or by modelbaker in the background).
```

Comment:
Can you transform this into a function description
This looks more like a commit message or a comment in the code,